test/contrib/automotive/xcp/xcp.uts

```diff
@@ -0,0 +1,823 @@
+% Regression tests for the XCP
+~ python3_only vcan_socket needs_root linux
```

Comment:
Does really the whole test require root?
And is the hole XCP protocol implementation python 3 only?